plugins/uefi-capsule/fu-uefi-capsule-plugin.c

```diff
@@ -676,7 +676,7 @@ fu_uefi_capsule_plugin_get_default_esp(FuPlugin *plugin, GError **error)
 
 			/* in case an installer messes up the kind */
 			id_type = fu_volume_get_id_type(esp);
-			if (g_strcmp0(id_type, "btrfs") == 0) {
+			if (g_strcmp0(id_type, "vfat") != 0) {
```

Comment:
I actually think we should fix this lower; I mean we're iterating the volumes from fu_context_get_esp_volumes() -- but we shouldn't have added the volume to this list in the first place. In `fu-context.c` we already deny-list `ext4` so it kinda points that we've seen this kind of thing before. Maybe we move the allowlist to `fu_context_get_esp_volumes()` and remove the ext4 entry there?

Reply:
done